qutebrowser/config/configfiles.py

```diff
@@ -262,6 +262,17 @@ def _handle_migrations(self, settings):
             del settings[old]
             self._mark_changed()
 
+        # window.hide_wayland_decoration was replaced by a more system agnostic
+        # Qt based equivalent
```

Comment:
Oh, damn - I forgot there was an easier way to tag renamed settings if there were no changes in the allowed values... Sorry for that! :blush: 

You should be able to remove this (and the associated test), and add this to the `configdata.yml` file:

```yaml
window.hide_wayland_decoration:
    renamed: ￼window.hide_decoration
```

That'll also display a nicer error message as qutebrowser knows what it was renamed to. Sorry again for the wasted time... my bad!

Reply:
No problem, your instructions are very clear and the amount of work done is pretty small!

Reply:
I do have to say that although an error message about the setting being renamed is now shown, having the old option present in a config file triggers multiple other errors and depending on further actions also does not get qutebrowser to start. This might be intentional/unavoidable however.

Reply:
Does it? When I have a `config.py` with `c.window.hide_wayland_decoration = True` I get:

> Errors occurred while reading config.py:
>
> - **While setting 'window.hide_wayland_decoration'**: No option 'window.hide_wayland_decoration' (this option was renamed to 'window.hide_decoration')

but I can still start qutebrowser as expected.

Reply:
nvm, it ended up working fine after a while for me to.